notification/test/notification/controllers/MainSpec.scala

```diff
@@ -48,20 +55,7 @@ class MainSpec(implicit ec: ExecutionEnv) extends PlaySpecification with Mockito
       val response = main.pushTopics()(request)
 
       status(response) must equalTo(CREATED)
-      there was one(reportRepository).store(expectedReport)
-    }
-
-    "notify reporting repository about added notifications and propagate reporting error" in new MainScope {
```

Comment:
Yes. If the reporting repository fails the notification will fail, since it needs the reporting repo for the de-duplication.